.github/workflows/system-tests.yml

```diff
@@ -17,17 +17,24 @@ jobs:
           cache: "yarn"
           cache-dependency-path: system-tests/yarn.lock
 
-      - name: Configure git to don't use unauthenticated protocol
+      - name: Configure git to not use unauthenticated protocol
         run: git config --global url."https://".insteadOf git://
 
+      - name: Cleanup for fresh installation
+        working-directory: ./solidity
+        run: |
+          rm -rf node_modules
+
       - name: Setup scenario prerequisites
         working-directory: ./solidity
         env:
           WALLET_PUBLIC_KEY: ${{ secrets.SYSTEM_TESTS_WALLET_PUBLIC_KEY }}
-          ETH_NETWORK: localhost
+          ETH_NETWORK: system_tests
         run: |
           yarn install
-          npx hardhat node --export export.json &
+          yarn clean
+          npx hardhat node --no-deploy &
+          USE_EXTERNAL_DEPLOY=true npx hardhat deploy --network $ETH_NETWORK --export export.json
```

Comment:
Shouldn't we update [the readme on system tests](https://github.com/keep-network/tbtc-v2/blob/main/system-tests/README.adoc).

Reply:
Good point [0cfa14f](https://github.com/keep-network/tbtc-v2/pull/421/commits/0cfa14f9276420ecb5248cab17d4320d2ce76c5b)